templates/plugins/importexport/resultsExport.tpl

```diff
@@ -23,6 +23,7 @@
 		<form id="exportIssuesXmlForm" class="pkp_form" action="{plugin_url path="downloadExportFile"}" method="post">
 			{csrf}
 			<input type="hidden" name="downloadFilePath" id="downloadFilePath" value="{$exportPath}" />
+			<input type="hidden" name="downloadFileName" id="downloadFileName" value="{$exportFileName}" />
```

Comment:
Both these hidden variables need to be passed through `|escape`.

Reply:
Fixed.